src/autopas/containers/cellTraversals/CellTraversal.h

```diff
@@ -10,29 +10,28 @@
 #include <array>
 #include <vector>
 
-#include "autopas/containers/TraversalInterface.h"
-
 namespace autopas {
 
 /**
  * A cell pair traversal.
  * This class handles traversals through the cell structures.
- * Derived classes handle the order through which the cells are traversed.
+ * Derived classes handle the order through which the cells are traversed and should additional inherit from
+ * TraversalInterface
  * @tparam ParticleCell type of cells.
  */
 template <class ParticleCell>
-class CellPairTraversal : public TraversalInterface {
+class CellTraversal {
```

Comment:
Because at least right now, `TraversalInterface` requires the interaction type as a template and the generic `CellTraversal` is independent from the interaction type and also doesn't require the methods from `TraversalInterface`